R/editing.R

```diff
@@ -233,7 +233,7 @@ update_object <- function(mn, pid, path, format_id=NULL, new_pid=NULL, sid=NULL)
 #' @param parent_data_pids (character)  Optional. Identifier for the data objects of the parent package. Not optional if the parent package contains data objects.
 #' @param parent_child_pids (character) Optional. Resource map identifier(s) of child packages in the parent package.  \code{resource_map_pid} should not be included. Not optional if the parent package contains other child packages.
 #' @param child_pids (character) Optional. Child packages resource map PIDs.
-#' @param metadata_path (character) Optional. Path to a metadata file to update with. If this is not set, the existing metadata document will be used.
+#' @param metadata_path (character or eml) Optional. An eml class object or a path to a metadata file to update with. If this is not set, the existing metadata document will be used.
```

Comment:
Can/should this arg name get changed now that it's not just a path?

Reply:
I didn't want to change the name so that it could remain backwards-compatible.  But if our preference is to make it friendlier for new users, then a name change makes sense

Reply:
Yeah that's probably more important.